python/doc/examples/meta_modeling/kriging_metamodel/plot_kriging_hyperparameters_optimization.py

```diff
@@ -88,38 +89,53 @@
 # --------------------
 
 # %%
-# In order to create the kriging metamodel, we first select a constant trend with the `ConstantBasisFactory` class. Then we use a squared exponential covariance model. Finally, we use the `KrigingAlgorithm` class to create the kriging metamodel, taking the training sample, the covariance model and the trend basis as input arguments. 
+# In order to create the kriging metamodel, we first select a constant trend with the `ConstantBasisFactory` class. Then we use a squared exponential covariance model. Finally, we use the `KrigingAlgorithm` class to create the kriging metamodel, taking the training sample, the covariance model and the trend basis as input arguments.
 
 # %%
 dimension = myDistribution.getDimension()
 basis = ot.ConstantBasisFactory(dimension).build()
-covarianceModel = ot.SquaredExponential([1.]*dimension, [1.0])
+
+# Trick B, v2
+x_range = X_train.getMax() - X_train.getMin()
+print("x_range:")
+print(x_range)
+scale_max_factor = 4.0  # Must be > 1, tune this to match your problem
+scale_min_factor = 0.1  # Must be < 1, tune this to match your problem
+maximum_scale_bounds = scale_max_factor * x_range
+minimum_scale_bounds = scale_min_factor * x_range
```

Comment:
I personally do not agree with this. I think that the minimum_scale_bounds value for each dimension should be proportional to the smallest distance within the data set for the given dimension, not the largest.
I have suggested an alternative initialization in the relative discussion on the Discourse channel ( https://openturns.discourse.group/t/normalization-of-the-input-sample-in-krigingalgorithm-in-ot1-16/101/2 )

Reply:
I do not agree :) This might be discussed more izili tomorrow at the TC.